CONTRIBUTING.md

```diff
@@ -0,0 +1,18 @@
+Contributing
+============
+
+Contributions are welcome from the community. Questions can be asked on the
+[issues page][1]. Before creating a new issue, please take a moment to search
+and make sure a similar issue does not already exist. If one does exist, you
+can comment (most simply even with just a `:+1:`) to show your support for that
+issue.
+
+If you have direct contributions you would like considered for incorporation
+into the project you can [fork this repository][2] and
+[submit a pull request][3] for review.
+
+
+
+[1]: https://github.com/USGS-R/drb-inland-salinity-ml/issues
```

Comment:
If we are transitioning this repo to GitLab and not DGEC, we'll have to remember to update these url's whenever that happens (here, and the url's in the code.json file).